Gems/LyShine/Code/Source/LyShineLoadScreen.cpp

```diff
@@ -141,6 +181,8 @@ namespace LyShine
     void LyShineLoadScreenComponent::UpdateAndRender([[maybe_unused]] float deltaTimeInSeconds)
     {
         AZ_Assert(m_isPlaying, "LyShineLoadScreenComponent should not be connected to LoadScreenUpdateNotificationBus while not playing");
+
+#if !defined (CARBONATED)
```

Comment:
Please reverse these
#if defined(CARBONATED)
..
#else
#endif

it helps to stay consistent.
(and same with other instances of !defined)